pixl_dcmd/src/pixl_dcmd/_database.py

```diff
@@ -62,6 +62,25 @@ def get_uniq_pseudo_study_uid_and_update_db(
         return UID(pseudo_study_uid, validation_mode=pydicom.config.RAISE)
 
 
+def get_pseudo_patient_id_and_update_db(
```

Comment:
This function should be called in the production code, and the pseudo_patient_id should be used to set the DICOM `dataset`'s patient tag. Like we do with updating the study uid in the dicom data

Reply:
[52475aa](https://github.com/UCLH-Foundry/PIXL/pull/484/commits/52475aa4614d753699d3dcfd59889072681aecb6)